src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SafeAreaView, ScrollView, StyleSheet, Text } from 'react-native'
+import { KeylessBackupEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import GoogleIcon from 'src/icons/Google'
+import Times from 'src/icons/Times'
+import { emptyHeader } from 'src/navigator/Headers'
+import { navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarIconButton } from 'src/navigator/TopBarButton'
+import fontStyles from 'src/styles/fonts'
+
+function onPressGoogle() {
+  ValoraAnalytics.track(KeylessBackupEvents.sign_in_with_google)
+  // TODO: start google sign-in
```

Comment:
This will be included in the same ticket, ACT-743, right?

Reply:
yes, will be in a follow up PR